packages/wdio-allure-reporter/README.md

```diff
@@ -1,7 +1,7 @@
 WDIO Allure Reporter
 ====================
 
-> A WebdriverIO reporter plugin to create [Allure Test Reports](https://docs.qameta.io/allure/).
+> A WebdriverIO reporter plugin to create [Allure Test Reports](https://qameta.io/allure-report/).
```

Comment:
Your link leads to Allure TestOps docs.
Should be https://docs.qameta.io/allure-report/